data/src/main/java/telegram4j/json/EncryptedPassportElementData.java

```diff
@@ -1,12 +1,16 @@
 package telegram4j.json;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 import org.immutables.value.Value;
 
 import java.util.List;
 import java.util.Optional;
 
 @Value.Immutable
+@JsonSerialize(as = EncryptedPassportElementData.class)
```

Comment:
Here should be `ImmutableEncryptedPassportElementData` class

Reply:
https://github.com/Telegram4J/Telegram4J/pull/1/commits/52586cc278dc16bd91a8c89a3c229f051b3712b0 :ok_hand: